docs/whats-new.asciidoc

```diff
@@ -4,112 +4,138 @@
 
-The new {security-guide}/alerts-ui-monitor.html#rule-execution-logs[*Rule execution logs*] tab on a rule's details page provides historical data for the rule's executions over time. Use this to understand how a particular rule is running and whether it’s creating the alerts you expect.
+*New Actions column in rule execution logs table enables filtering*
+
+You can create a {security-guide}/alerts-ui-monitor.html#rule-execution-logs[global search filter] based on a specific rule execution by selecting the filter icon in the *Actions* column of the *Rule execution logs* tab on the rule details page. Enabling this filter replaces any previously applied filters.
 
 [role="screenshot"]
-image::whats-new/images/8.2/rule-exec-logs.png[]
+image::whats-new/images/8.3/actions-icon.png[]
 
-[discrete]
-=== Bulk apply a Timeline template to rules
+*New prebuilt rules*
 
-A new {security-guide}/rules-ui-management.html#edit-rules-settings[bulk actions option] allows you to apply a Timeline template to multiple rules at once.
+15 new {security-guide}/prebuilt-rules.html[prebuilt rules] have been added in this release.
 
 [discrete]
-=== New Rules table filter options
-
-You can now {security-guide}/rules-ui-management.html#sort-filter-rules[filter the Rules table] by index pattern, MITRE ATT&CK tactic or technique (name or ID), and rule name.
+== OAuth support in ServiceNow connectors
 
-*Rule preview feature includes alerts table*
+The ServiceNow connectors now support open authentication (OAuth).
+For configuration details, refer to
+{kibana-ref}/servicenow-action-type.html[ServiceNow ITSM], {kibana-ref}/servicenow-sir-action-type.html[ServiceNow SecOps],
+and {kibana-ref}/servicenow-itom-action-type.html[ServiceNow ITOM connector].
 
-When you create or edit a detection rule and preview it, the {security-guide}/rules-ui-create.html#preview-rules[rule preview] now includes an alerts table with the expected alerts for the rule. Use this feature to learn how noisy a rule may be before saving it. You can now also preview {security-guide}/rules-ui-create.html#create-indicator-rule[indicator match rules].
+[discrete]
+== Cases enhancements
 
-[role="screenshot"]
-image::whats-new/images/8.2/preview-rules.png[]
+The following enhancements have been added to Cases:
 
-*Turn off `read` privilege warnings for detection rules*
+* You can now can assign a severity level to cases.
++
+NOTE: If you do not set the case's severity, it defaults to Low.
++
+* The Cases table now includes a *Severity* column and an option to filter the table by severity. The average time to close case metrics has also been added to the Cases table.
+* You can now delete text comments, including those in Lens visualizations.
+* You can add multiple alerts to new and existing cases through the *Bulk actions* menu.
```

Comment:
```suggestion
* You can now add multiple alerts to new and existing cases via the *Bulk actions* menu.
```

Reply:
@benironside per UX guidelines, we are doing away with `via`.